contributions/executable-tutorial/mrhasa-ved.megha/README.md

```diff
@@ -0,0 +1,18 @@
+# Executable Tutorial: Automatic testing of the application using Selenium
+
+### Members
+
+Megha Gangwal (ved.megha@gmail.com)
+GitHub: [gangwalmegha](https://github.com/gangwalmegha/)
```

Comment:
Provide a KTH email.
Also, the folder name should be "kth_id1-kth_id2", please fix it.

Reply:
Hi,
I do not have the KTH email-id. I am taking this course as an independent course

Reply:
That is fine, you can follow the course, but we cannot merge a PR from a student who does not have a KTH id. Please try to register for a KTH id as soon as possible.

I'm going to close this PR, but your teammate can create a new PR with only his/her own KTH id in the contribution folder name.